test/unit/bin-tessel-2.js

```diff
@@ -87,12 +88,11 @@ exports['Tessel (cli: restart)'] = {
   noError: function(test) {
     test.expect(1);
 
-    cli(['restart']);
+    cli(['restart', '--type=ram']);
```

Comment:
For some reason, the `restart` command doesn't get the default `type` property assigned to it by `nomnom` in the unit tests. The default param shows up properly w/o tests.